dev/AppNotifications/WICUtility.cpp

```diff
@@ -0,0 +1,166 @@
+﻿
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "WICUtility.h"
+#include <cwctype>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/base.h>
+#include <externs.h>
+
+#include <ShObjIdl_core.h>
+#include <wincodec.h>
+#include <Ocidl.h>
+#include <windows.h>
+
+typedef enum
+{
+    BMPV_1,
+    BMPV_5,
+} BITMAP_VERSION;
+
+winrt::com_ptr<IWICBitmapSource> ConvertWICBitmapPixelFormat(
```

Comment:
It sounds like you had to read too much code before putting all of it here. Thanks for looking into it.